systemd/hyprland-session.service.in

```diff
@@ -9,8 +9,8 @@ After=graphical-session-pre.target
 
 [Service]
 Type=notify
-ExecStart=/usr/bin/Hyprland
-ExecStop=/usr/bin/hyprctl dispatch exit
-ExecStopPost=/usr/bin/systemctl --user unset-environment DISPLAY WAYLAND_DISPLAY HYPRLAND_INSTANCE_SIGNATURE XDG_CURRENT_DESKTOP
+ExecStart=@CMAKE_INSTALL_PREFIX@@prefix@/bin/Hyprland
```

Comment:
Could be easier to only leave `@bindir@/Hyprland` and use `set(bindir ${CMAKE_INSTALL_BINDIR})` in CMakeLists.txt.

Reply:
for some reason, both bindir from meson and CMAKE_INSTALL_BINDIR default to just /bin

Reply:
ah yea, then use `@prefix@/@bindir@`

Reply:
Okay, workin on it

Reply:
done